etc/checkstyle-suppressions.xml

```diff
@@ -57,6 +57,10 @@
     <suppress checks="IllegalImport"
             files="DerUtils\.java"/>
 
+    <!-- Metrics Prometheus output generation depends on numerous settings -->
+    <suppress checks="ParameterNumber"
+              files="MetricImpl\.java"/>
+
```

Comment:
Checkstyle exclusion needs to be confirmed by release manager @barchetta

Reply:
LGTM